apps/web/src/components/ProfileFormWrapper/ProfileFormWrapper.tsx

```diff
@@ -38,8 +61,16 @@ const ProfileFormWrapper = ({
     links = [
       {
         label: intl.formatMessage({
-          defaultMessage: "My profile",
-          id: "+lKz3l",
+          defaultMessage: "Home",
+          id: "EBmWyo",
+          description: "Link text for the home link in breadcrumbs.",
+        }),
+        url: paths.dashboard(),
+      },
+      {
+        label: intl.formatMessage({
+          defaultMessage: "Profile and applications",
+          id: "WVbN0f",
           description: "Breadcrumb from applicant profile wrapper.",
         }),
         url: user?.id ? paths.profile(user.id) : paths.myProfile(),
```

Comment:
Not part of this but, I think we will need to update this path when we removed the `APPLICANT_DASHBOARD` flag. Using breadcrumbs was disorienting 😵‍💫

Reply:
Oh, I probably have these wrong!  Should be:

Home -> App Homepage
Profile and applications -> Applicant Dashboard

@substrae could you confirm?

Reply:
Looks like your correction above is right - `Home` goes to talent.canada.ca, `Profile and applications` takes them to what we were previously referring to as their dashboard.